.travis.yml

```diff
@@ -1,16 +1,27 @@
-language: python
+language: php
 
-python:
-    - "2.7"
+php: 5.6
 
-sudo: false
+sudo: true
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+    - _build
 
-script: sphinx-build -nW -b html -d _build/doctrees . _build/html
+before_install:
+  - python --version
+  - sudo pip install --upgrade pip
+
+install:
+  - sudo pip install sphinx==1.1.3
+  - composer require wouterj/docbot:~0.2@dev
```

Comment:
should docbot really be installed as a dependency ? This looks weird to me. It is a project, which should use its own locked dependencies IMO (what about a phar distribution for it?).

Btw, the sphinx build currently does not ignore the vendor folder, causing issues as there are some rST files in it.